archetype/simple-service/src/main/resources/archetype-resources/pom.xml

```diff
@@ -82,6 +82,11 @@
                 <version>2.0.2</version>
             </dependency>
             <dependency>
+                <groupId>com.google.guava</groupId>
+                <artifactId>guava</artifactId>
+                <version>15.0</version>
```

Comment:
Is there anything keeping us from going higher with this? I think `21` is the latest.